lib/netext/httpext/request.go

```diff
@@ -36,11 +33,9 @@ import (
 	"strings"
 	"time"
 
-	ntlmssp "github.com/Azure/go-ntlmssp"
-	"github.com/andybalholm/brotli"
-	"github.com/klauspost/compress/zstd"
+	"github.com/Azure/go-ntlmssp"
 	"github.com/sirupsen/logrus"
-	null "gopkg.in/guregu/null.v3"
```

Comment:
I am wondering why this and the above ntlmssp changed ...

Reply:
Seems it's the different version of `goimports` in my Mac and Linux machine.